day_10/task_1/tests/pipe_maze_tests.cpp

```diff
@@ -58,3 +59,19 @@ TEST_F(PipeMazeTests, GivenStartingPoint_WhenGoForwardTwoTimesInSecondDirection_
         go_forward(pipe_maze, current_point_in_second_direction, foundNeighbors.second);
     ASSERT_EQ(current_point_in_second_direction, MazePoint(3, 2));
 }
+
+TEST_F(PipeMazeTests, GivenStartingPoint_WhenCountStepsToFarthestPointFromBasicPipeMaze_ShouldReturnNumberOfStepsEuqalFour)
+{
+    const MazePoint starting_pipe(find_starting_point(pipe_maze));
```

Comment:
Do we really need pass starting point as second argument?
Maybe `pipe_maze` is enough, because we can run `find_starting_point` on `pipe_maze` internally?

Reply:
If you agree, then remember to correct the description of test as well.

Reply:
done